ui/ui-frontend/projects/archive-search/src/app/archive/archive-search/additional-actions-search/management-rules/management-rules.component.html

```diff
@@ -43,23 +43,19 @@ <h5>{{ 'ARCHIVE_SEARCH.RULES_ACTION.UPDATE_RULE' | translate }}</h5>
           </div>
         </div>
         <div class="col-8 row-padding">
-          <div class="update-rules-table-head row">
-            <div class="col-2 clickable" (click)="returnToArchiveSearchPage()">
-              <span class="text primary"> {{ 'RULES.BACK' | translate }}</span>
+          <div class="update-rules-table-head">
+            <div class="col-6">
+              <span class="details-ua">
+                {{ 'RULES.UA_SELECTED' | translate: { ua_selected: selectedItemToShow } }}
+              </span>
             </div>
-            <div class="col-10">
-              <div class="row">
-                <div class="col-7">
-                  <span class="details-ua">
-                    {{ 'RULES.UA_SELECTED' | translate: { ua_selected: selectedItemToShow } }}
-                  </span>
-                </div>
-                <div class="col-5">
-                  <button class="btn primary" [disabled]="ruleActionsAbsentOrInvalid()" (click)="submitUpdates()">
-                    {{ 'RULES.EXECUTE_RULE_UPDATE' | translate }}
-                  </button>
-                </div>
-              </div>
+            <div class="col-4">
+              <button class="btn primary" [disabled]="ruleActionsAbsentOrInvalid()" (click)="submitUpdates()">
+                {{ 'RULES.EXECUTE_RULE_UPDATE' | translate }}
+              </button>
+            </div>
+            <div class="col-2 btn outline outline-primary" (click)="returnToArchiveSearchPage()">
+              <span class="text primary"> {{ 'RULES.QUIT' | translate }}</span>
             </div>
           </div>
```

Comment:
KO en 1440px

<img width="766" height="174" alt="Image" src="https://github.com/user-attachments/assets/c9411a0d-9de9-47b9-abce-7d2a64663583" />

```suggestion
<div class="update-rules-table-head justify-content-between">
            <span>
              {{ 'RULES.UA_SELECTED' | translate: { ua_selected: selectedItemToShow } }}
            </span>
            <div class="hgap-3">
              <button class="btn primary" [disabled]="ruleActionsAbsentOrInvalid()" (click)="submitUpdates()">
                {{ 'RULES.EXECUTE_RULE_UPDATE' | translate }}
              </button>
              <button class="btn outline outline-primary" (click)="returnToArchiveSearchPage()">
                {{ 'RULES.QUIT' | translate }}
              </button>
            </div>
          </div>
```

Et supprimer le style "details-ua"

Reply:
c'est fait!